packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -87,7 +90,10 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
       browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-default');
       browser.refresh();
       Terra.hideInputCaret('input[name="terra-time-minute-input"]');
-      browser.setValue('input[name="terra-date-input"]', '03/11/2018');
+      browser.click('input[name="terra-date-month-input"]');
```

Comment:
If our tests need changed in this way, I am worried about the passivity of these changes for our consumers that have written tests for this component.

Reply:
It definitely wouldn't be passive in that case, just by virtue of the one input being split into 3.